src/main/java/ai/elimu/util/epub/EPubParagraphExtractionHelper.java

```diff
@@ -119,6 +119,61 @@
                             }
                         }
                         
+                        // Look for paragraphs within `<div lang="en">` (StoryBookProvider#LETS_READ_ASIA)
+                        // Expected format:
+                        /*
+                            <div lang="en">
+                                <div class="container">
+                                    <img src="p-1.jpg" alt=""/>
+                                </div>
+                                <p dir="auto">The moon rises.</p>
+                            </div>
+                        */
+                        if ("div".equals(bodyChildNode.getNodeName()) && (bodyChildNode.getAttributes().getNamedItem("lang") != null)) {
+                            NodeList langDivChildNodeList = bodyChildNode.getChildNodes();
+                            logger.info("langDivChildNodeList: " + langDivChildNodeList);
```

Comment:
To avoid mixing with variable names already used above, consider renaming from "child node" to "paragraph container" or something similar?